### ISSUE TITLE
Color contrast does not reach WCAG AA Accessibility standards

### ISSUE THREAD

**Issue by elioqoshi** [gembarrett/soap#57]
Firstly, great work on SOAP!

I wanted to notice that the color contrast ratio of white on light blue does not reach WCAG AA. That means that it's hardly readable at points, especially for people with vision disabilities.

![image](https://user-images.githubusercontent.com/5436686/69903699-5bd99200-139d-11ea-804a-c5bbaadf3594.png)

Currently the ratio is 1.85. It should be at least 4.5 to pass WCAG AA. That would be possible by making one of these changes:

- Change the light blue to a darker shade which reaches a contrast ratio of 1:4.5 or more
- Change the text color of elements on the light blue to a shade of dark blue in order to reach a contrast ratio of 1:4.5

It would be great to see this being addressed. Happy to help with making this change as well :)

If there is any opportunity to address that,

**Comment by gembarrett** [comment]
Thanks so much for bringing this up, it's true that I need to do more about the accessibility of SOAP and the text colours are a great place to start. I will get this sorted asap and report back here when it's done.

Also, sorry for not responding sooner, for some reason I didn't get a notification.

**Comment by gembarrett** [comment]
Note to self: #668EAA dark blue used elsewhere still only registers as 3.49 (https://contrast-ratio.com/#white-on-%23668EAA) so need to find another alternative

**Comment by elioqoshi** [comment]
@gembarrett Happy to help :)

A nice tool which I like using for context is also: https://whocanuse.com/

If you want to avoid spending time creating a color palette from scratch I'd suggest having a look at Mineral UI: https://mineral-ui.netlify.com/color

Alternatively, GitLab uses accessibile color palettes: https://design.gitlab.com/product-foundations/colors

Hope that helps :)

**Comment by gembarrett** [comment]
Ah that's a great help, thanks!

**Comment by gembarrett** [comment]
Ok, so this change prompted a redesign of the site :) It'll be in the beta release next month. Thanks for pointing this issue out @elioqoshi!

**Comment by elioqoshi** [comment]
@gembarrett Happy to! If you need more help, throughout OTF Learning Lab it should be something I can help with as well :)
